task_Dev_2/calculator/Calculator.java

```diff
@@ -0,0 +1,88 @@
+package task_Dev_2.calculator;
+
+import java.util.*;
+
+/**
+ * Calculate expression stored in list.
+ */
+class Calculator {
+  private static final String DIVISION = "/";
+  private static final String MULTIPLICATION = "*";
+  private static final String SUBTRACTION = "-";
+  private static final String ADDITION = "+";
+  private static final String FORMAT_EXCEPTION_MESSAGE = "Incorrect number format! ";
+  
+  /**
+   * Calculate results by priority of operations.
+   *
+   * @param exprArr - list with operands and operators
+   * @exception NumberFormatException Occurs when the operation of the incorrect number.
+   */
+  public void calculateResult(List<String> exprArr) {
+    try {
+      reWriteList(MULTIPLICATION, DIVISION, exprArr);
+      reWriteList(ADDITION, SUBTRACTION, exprArr);
+      if (exprArr.size() == 1) {
+        System.out.println(exprArr.get(0));
+      }
+    } catch (NumberFormatException ex) {
```

Comment:
You can't do anything except logging here so probably  it's not appropriate place to catch exception